agent/.env.development

```diff
@@ -1,5 +1,3 @@
 AGENT_WALLET_KEY=secret-wallet-key
-DID_INDY_INDICIO_TESTNET_PUBLIC_DID_SEED=2543786a945a27258087ccfe95ff62df
-CHEQD_TESTNET_COSMOS_PAYER_SEED=robust across amount corn curve panther opera wish toe ring bleak empower wreck party abstract glad average muffin picnic jar squeeze annual long aunt
 ED25519_SEED=5473a3e4c5ae3fd5fb3ad089563596e3
-P256_SEED=e5f18b10cd15cdb76818bc6ae8b71eb475e6eac76875ed085d3962239bbcf42f
\ No newline at end of file
+P256_SEED=e5f18b10cd15cdb76818bc6ae8b71eb475e6eac76875ed085d3962239bbcf42f
```

Comment:
It might be beneficial to add the root certificate as an env variable. As we have sqlite storage and it will run in a container. So we either have to set up postgres database, or we should generate the certificate once and pass it as variable, so that we can re-use it.

Otherwise we have to update the trusted root certificate every time in the wallet.

Reply:
I set the input to make sure it is always the same certificate for the same host. This way when we restart the docker container it will just recreate the exact same certificate (key, notBefore and notAfter claims are always the same).